src/__init__.py

```diff
@@ -119,14 +155,88 @@ def _anyWindowMinimized(self):
             w.windowState() == Qt.WindowState.WindowMinimized
             for w in self._visibleWindows()
         )
+        
+    def _createReviewsIcon(self, string, renderNumber = True):
+        pixmap = QPixmap("icons:anki.png")
+
+        textColor = theme_manager.qcolor(colors.STATE_LEARN)
+        backgroundColor = theme_manager.qcolor(colors.CANVAS_ELEVATED)
+
+        fontSize = self.dueFontSize
+        sizeRectF = QRectF(0, 32 - fontSize, 32, fontSize)
+        sizeRect = QRect(0, 32 - fontSize, 32, fontSize)
+
+        if renderNumber:
+            painter = QPainter(pixmap)
 
-    def _createTrayIcon(self):
+            # Draw number container
+
+            roundedRectanglePath = QPainterPath()
+            roundedRectanglePath.addRoundedRect(sizeRectF, 5, 5)
+
+            painter.fillPath(roundedRectanglePath, backgroundColor)
+
+            # Draw number
+
+            font = painter.font()
+            font.setPixelSize(fontSize)
+            font.setWeight(QFont.Weight.Bold)
+            painter.setFont(font)
+
+            painter.setPen(textColor)
+            painter.drawText(sizeRect, Qt.AlignmentFlag.AlignCenter, string)
+
+            painter.end()
+        
+        icon = QIcon()
+        icon.addPixmap(pixmap, QIcon.Mode.Normal, QIcon.State.Off)
+        return icon
+
+    def _getAmountOfCardsDue(self):
+
+        if not self.showDueAmount:
```

Comment:
This method should not check for `showDueAmount`. This check should be performed outside by some of the callers.